pkg/annotation/capa.go

```diff
@@ -0,0 +1,23 @@
+package annotation
+
+// Keep the YAML documentation format as it is used to render in the CRD public documentation. You can use Markdown in the documentation field.
+
+// AWSVPCMode is set on AWSCluster. Possible values are AWSVPCModePublic and AWSVPCModePrivate
+const AWSVPCMode = "aws.giantswarm.io/vpc-mode"
+
+// AWSVPCModePublic is one possible AWSVPCMode annotation value. When it is set,
+// CAPA controllers will create public-facing VPC for AWS cluster.
+const AWSVPCModePublic = "public"
```

Comment:
Do we have a convention to differentiate between annotation names and values?

Reply:
no that I'm aware.

Reply:
Yeah, looks like this is the only other example in this repo: https://github.com/giantswarm/k8smetadata/blob/main/pkg/label/general.go#L8